src/libecalc/presentation/yaml/yaml_types/emitters/yaml_venting_emitter.py

```diff
@@ -80,10 +84,29 @@ def check_user_defined_category(cls, category, info: ValidationInfo):
                     entity_name = str(cls)
 
                 raise ValueError(
+                    f"CATEGORY: {category} is not allowed for {cls.Config.title} {name}. Valid categories are: "
```

Comment:
cls.Config.title will not work anymore as we have upgraded to pydantic v2. model_config is the property we are using, but it does not seem to be set here